context.go

```diff
@@ -439,11 +439,6 @@ func (c *Context) GetPostFormArray(key string) ([]string, bool) {
 	if values := req.PostForm[key]; len(values) > 0 {
 		return values, true
 	}
-	if req.MultipartForm != nil && req.MultipartForm.File != nil {
-		if values := req.MultipartForm.Value[key]; len(values) > 0 {
-			return values, true
-		}
-	}
```

Comment:
Why remove this condition?

Reply:
Add this code for support go1.6, please commit message: https://github.com/gin-gonic/gin/pull/1383/commits/51386de165ae50a61c06d80f06ec703810a2d62d